### PR TITLE
Feat/refactor 09 2022

### PR DESCRIPTION
**Pull request by slaveatanasov**

### CODE REVIEW

File: src/components/universaldot/DAO/OrganizationCreateAndUpdateForm.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  vision: string;
+};
+
+type Props = {
+  form: FormType;
+  onCancel: VoidFunction;
+  organizationId?: string;
+};
+
+export default function OrganizationCreateAndUpdateForm({ form, onCancel, organizationId }: Props) {
```
Comment (stojanov-igor) [inline src/components/universaldot/DAO/OrganizationCreateAndUpdateForm.tsx:51]: Got the following warning here: 

`Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at OrganizationCreateAndUpdateForm `

Comment (stojanov-igor) [reply src/components/universaldot/DAO/OrganizationCreateAndUpdateForm.tsx:51]: Also this... 

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'signAndSend')
    at signedTx (useDao.ts:360:1)`

File: src/components/universaldot/DAO/OrganizationCreateAndUpdateForm.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  const { enqueueSnackbar } = useSnackbar();
```
Comment (stojanov-igor) [inline src/components/universaldot/DAO/OrganizationCreateAndUpdateForm.tsx:52]: What is a Snackbar? Sounds delicious. 🍡

File: src/components/universaldot/DAO/OrganizationTransferOwnershipForm.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  newOwnerId: string;
+};
+
+type Props = {
+  form: FormType;
+  onCancel: VoidFunction;
+  organizationId: string;
+};
+
+export default function OrganizationTransferOwnershipForm({
```
Comment (stojanov-igor) [inline src/components/universaldot/DAO/OrganizationTransferOwnershipForm.tsx:45]: Got this error when transferring ownershiop also.. 
`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'signAndSend')
    at signedTx (useDao.ts:360:1)`

**Review by stojanov-igor — CHANGES_REQUESTED** [review]

File: src/components/universaldot/DAO/OrganizationTransferOwnershipForm.tsx (hunk 3 above)
Comment (slaveatanasov) [reply src/components/universaldot/DAO/OrganizationTransferOwnershipForm.tsx:45]: We need to test it together.

**Review by stojanov-igor — APPROVED** [review]

**Pull request merged**